app-cdr/cdrdao/cdrdao-1.2.4-r2.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic
+
+MY_PV="rel_${PV//./_}"
+DESCRIPTION="Burn CDs in disk-at-once mode -- with optional GUI frontend"
+HOMEPAGE="https://github.com/cdrdao/cdrdao/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/refs/tags/${MY_PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="encode mad vorbis"
+
+DEPEND="app-cdr/cdrtools
+	encode? ( >=media-sound/lame-3.99 )
+	mad? (
+		media-libs/libao
+		media-libs/libmad
+	)
+	vorbis? (
+		media-libs/libao
+		media-libs/libvorbis
+	)"
+RDEPEND="${DEPEND}
+	!app-cdr/cue2toc"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}/${P}-ax_pthread.patch"
+	"${FILESDIR}/${P}-wformat-security.patch"
+)
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+src_prepare() {
+	cd "${S}" || die
+	sed -i -e "s|^AM_GCONF_SOURCE_2||" configure.ac # https://bugs.gentoo.org/867235
```

Comment:
This should be a patch, not a sed.

Reply:
patch or sed - one maintainer wants patch, the other one is fine with a sed....
A patch would look strange here, because it must be `eapply`ed before autogen,sh, which must run before `default`...
The order seems to be important here, I was working around this the last 2 hours.

Ok, I must stop for now here, will continue in the evening.